### ISSUE TITLE
can you share the training log?

### ISSUE THREAD

**Issue by Feynman1999** [HVision-NKU/SRFormer#15]
In the process of reproducing, taking the light weight SR task as an example, it can reach around 38.0db on set5, which is still 0.2db away from the paper?If there is a log, it will help me

**Comment by Z-YuPeng** [comment]
It might be something wrong. In my memory, the performance will exceed SwinIR within 200k iters of the total 500k training process. Could you provide more information about your reproducing? Are the patch size(64x64), batch size(64), and dataset(DIV2K 800 images) the same as our setting?

**Comment by Feynman1999** [comment]
Since I have only one GPU, I used batch size 16 and learning rate 2e-4. The dataset and patch size are consistent with the paper. Additionally, I employed the CosineAnnealingWarmRestarts scheduler with a total of 600,000 iterations, T=100,000, and T_multi=1. So, there are indeed some differences, but I don't think they would result in a 0.2dB gap. I want to figure out where the issue lies. I will try to align the parameters as much as possible, but I cannot achieve a batch size of 64. If possible, it would be great if you could train with a batch size of 16.

**Comment by Feynman1999** [comment]
Sorry, I forgot to mention that I can use your code directly to run a batch size of 16, and then compare it with my code. Once I have the results, I will update them here.

**Comment by Z-YuPeng** [comment]
Have you previously utilized the identical configuration to train a SwinIR?

**Comment by Feynman1999** [comment]
> Have you previously utilized the identical configuration to train a SwinIR?

not yet.  thank you for your advice, I will try it .
